OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -392,6 +393,10 @@ public String getOAuthClientId() {
         return getString(OAUTH_CLIENT_ID);
     }
 
+    public String getOrganizationCode() {
+        return getString(ORGANIZATION_CODE);
```

Comment:
You can set a default here. That way we won't have to add a blank config entry in the default_config file.